docs/Gemfile.lock

```diff
@@ -101,10 +101,10 @@ GEM
     middleman-syntax (3.0.0)
       middleman-core (>= 3.2)
       rouge (~> 2.0)
-    mini_portile2 (2.2.0)
+    mini_portile2 (2.3.0)
     minitest (5.10.1)
-    nokogiri (1.8.0)
-      mini_portile2 (~> 2.2.0)
+    nokogiri (1.8.1)
+      mini_portile2 (~> 2.3.0)
```

Comment:
@budhrg Why we need to update the version of `mini_portile2` and how did you reach to that particular version?

Reply:
@LalatenduMohanty  See https://ci.centos.org/job/minishift-pr/3210/console 
```
13:48:18 Installing mini_portile2 2.2.0
13:48:18 Fetching nokogiri 1.8.1
13:48:19 Downloading nokogiri-1.8.1 revealed dependencies not in the API or the lockfile
13:48:19 (mini_portile2 (~> 2.3.0)).
13:48:19 Either installing with `--full-index` or running `bundle update nokogiri` should
13:48:19 fix the problem.
13:48:19 
13:48:19 In Gemfile:
13:48:19   html-proofer was resolved to 3.7.2, which depends on
13:48:19     nokogiri
```

Actually, the ideal way is `bundle update nokogiri` command which checks if its dependent gems is also compatible or not.

Reply:
@LalatenduMohanty Seems minimum requirement of `mini_portfile2` for `nokogiri 1.8.1` change to `2.3.0`  See https://rubygems.org/gems/nokogiri/versions/1.8.1